configuration/example-config.json

```diff
@@ -23,7 +23,7 @@
   ],
   "Sinks": [
     {
-      "PackageRE": "k?log$",
```

Comment:
Did we want that terminal anchor?  Will that not overlook, e.g., `klog/v1`?

Reply:
I think we indeed do not want the `$`, since it does overlook `klog/v1`.

Reply:
Oh.  Monday mornings.  I was reading this diff absolutely backwards, like you were adding the anchor.